src/content/app/genome-browser/components/browser-sidebar-modal/modal-views/NavigateModal.tsx

```diff
@@ -0,0 +1,60 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import {
+  Accordion,
+  AccordionItem,
+  AccordionItemButton,
+  AccordionItemHeading,
+  AccordionItemPanel
+} from 'src/shared/components/accordion';
+import NavigateRegionModal from './navigate-modal/NavigateRegionModal';
+import NavigateLocationModal from './navigate-modal/NavigateLocationModal';
+
+import styles from './navigate-modal/NavigateModal.scss';
+
+const NavigateModal = () => {
+  return (
+    <div className={styles.accordionContainer}>
+      <Accordion className={styles.navigateAccordion}>
+        <AccordionItem className={styles.navigateAccordionItem}>
+          <AccordionItemHeading>
+            <AccordionItemButton className={styles.navigateAccordionItemButton}>
+              Navigate this region
+            </AccordionItemButton>
+          </AccordionItemHeading>
+          <AccordionItemPanel className={styles.navigateAccordionPanel}>
+            <NavigateRegionModal />
```

Comment:
Since this component (as well as `NavigateLocationModal` below) is no longer a modal, we should rename them. `GenomeBrowserRegionNavigation` and `GenomeBrowserLocationNavigation`?

Reply:
Can we just have it as `LocationNavigation` removing the `GenomeBrowser` prefix?

Reply:
We could 🤷‍♂️  The GenomeBrowser prefix would help with searching files by name in case we create other `RegionNavigation` or `LocationNavigation` components in other parts of the site; but I don't particularly mind if we skip it.

Reply:
The only reason is that none of the sidebar components have that prefix. Though one says BrowserSidebar :-( 
Region navigation is tied to genome browser, nowhere else it can be used I think

Reply:
But then we got the `GenomeBrowserNavigationButtons` Changing it to NavigationButtons is not a good idea. May be I will go back to your choice.

Reply:
Renamed all and removed prefix at the end, to avoid very long file names